docs/1.18.x/setup/configuration/file-structure.md

```diff
@@ -0,0 +1,39 @@
+# Setup - Configuration
+## Folder Naming
+
+The usual convention for folders is `com/yourName/modName`, but of course you don't have to stick to it.
+
+Each modder has their own style of folder naming that is very similar, for each content package, such as your code for blocks, I would suggest either `block` or `blocks`. This goes in your `common` package (read below)
```

Comment:
This section doesn't make too much sense when reading. Elaborate on the folder structure rather than stuff that will be mentioned next